### DISCUSSION TITLE
Grid Pagination customization

### DISCUSSION THREAD

**Discussion by mightygriffon6** [vikramlearning/blazorbootstrap#824] [Q&A]
![image](https://github.com/user-attachments/assets/9f3c4aaf-a4d9-4dc8-b978-b884ca3ef458)
Im using grid's pagination but i want to change some of the styles, like allignment or color

**Comment by gvreddy04** [comment]
@mightygriffon6 Thank you for using BlazorBootstrap. At this moment, customization is not supported. Please create an issue out of this. We will add it in the upcoming releases. Please share the different alignment combinations you are looking for. This will help me understand the requirement.
